database/index.ts

```diff
@@ -0,0 +1,50 @@
+import pgPromise = require('pg-promise'); // pg-promise core library
+import process from 'process';
+import {IInitOptions, IDatabase, IMain} from 'pg-promise';
+import {IExtensions, TCDocumentRepository, TCUserAgreementRepository} from './repos';
+import {Diagnostics} from './diagnostics';
+
+type ExtendedProtocol = IDatabase<IExtensions> & IExtensions;
+
+// pg-promise initialization options:
+const initOptions: IInitOptions<IExtensions> = {
+
+    // Extending the database protocol with our custom repositories;
+    // API: http://vitaly-t.github.io/pg-promise/global.html#event:extend
+    extend(obj: ExtendedProtocol, dc: any) {
+        // Database Context (dc) is mainly needed for extending multiple databases with different access API.
+
+        // Do not use 'require()' here, because this event occurs for every task and transaction being executed,
+        // which should be as fast as possible.
+        obj.documents = new TCDocumentRepository(obj, pgp);
+        obj.userAgreements = new TCUserAgreementRepository(obj, pgp);
+    }
+};
+
+/*const dbConfig = {
+    host: process.env.POSTGRES_HOST,
+    port: process.env.POSTGRES_HOST,
+    database: process.env.POSTGRES_HOST,
+    user: process.env.POSTGRES_HOST,
+    password: process.env.POSTGRES_HOST
+};*/
```

Comment:
Any reason to keep commented out code?

Reply:
Yeah, we're going to be using that instead of the hardcoded values